opencti-platform/opencti-front/package.json

```diff
@@ -20,6 +20,7 @@
     "apexcharts": "3.42.0",
     "axios": "1.5.0",
     "ckeditor5-custom-build": "0.0.1",
+    "classnames": "2.3.2",
```

Comment:
What is the purpose of that lib ?

Reply:
This lib is great to handle complex usages of classname in components.
It allows to have multiple classes or conditionnal classes in a component.